episodes/02-filedir.md

```diff
@@ -405,6 +405,35 @@ Note that if a directory named `Desktop` does not exist in your current working
 this command will return an error. Typically, a `Desktop` directory exists in your
 home directory, which we assume is the current working directory of your bash shell.
 
+:::::::::::::: callout
+
+### OneDrive Variations
+
+If a Windows machine is backed up with OneDrive, your Desktop may
+  be rendered from a folder within OneDrive, which will not match the contents of `~/Desktop`.
+  The OneDrive desktop should be accessible using one of the following commands
+  (if the name of the enterprise/org isn't clear, look through the output of `ls` to find
+  the right folder):
+  
+  ```bash
+  $ cd "~/OneDrive - Name Of Enterprise/Desktop"
+  $ cd "C:/Users/Username/OneDrive - Name Of Enterprise/Desktop"
+  ```
+
+Note: The quotation marks are needed since the Unix shell interprets spaces to be meaningful separators between files/commands/options.
```

Comment:
```suggestion
Note: The quotation marks are needed for this folder because the Unix shell interprets spaces to be meaningful separators between files/commands/options but this folder has spaces in the name.
```